modules/caddyhttp/server.go

```diff
@@ -732,4 +799,7 @@ const (
 	// For a partial copy of the unmodified request that
 	// originally came into the server's entry handler
 	OriginalRequestCtxKey caddy.CtxKey = "original_request"
+
+	// For tracking whether the client is a trusted proxy
+	TrustedProxyCtxKey caddy.CtxKey = "trusted_proxy"
```

Comment:
I wonder if this should just be stored as a var, that way it could be used by placeholders or CEL expressions, etc.

Reply:
Sure - I'm never sure which one is better to use 🤷‍♂️ a var would be fine 👍

Reply:
Alright, updated to use a var. We get replacer support out of the box 👍